cmake/CMakeLists.txt

```diff
@@ -1,15 +1,18 @@
 # Minimum CMake required
 cmake_minimum_required(VERSION 2.8.12)
 
-# Project
-project(protobuf C CXX)
-
 # CMake policies
 cmake_policy(SET CMP0022 NEW)
 
+# Project
+project(protobuf C CXX)
+
 # Options
 option(protobuf_VERBOSE "Enable for verbose output" OFF)
+option(protobuf_TARGETS_ALIASES "Provide targets aliases without protobuf:: namespace" ON)
```

Comment:
This should not be allowed, or at the very least should default to off. This has nothing to do with backwards compatibility, and is counter to CMake guidelines.